sdk/src/client/constants.rs

```diff
@@ -22,6 +22,9 @@ pub(crate) const DEFAULT_USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/",
 pub(crate) const MAX_PARALLEL_API_REQUESTS: usize = 100;
 /// Max allowed difference between the local time and latest milestone time, 5 minutes in seconds
 pub(crate) const FIVE_MINUTES_IN_SECONDS: u32 = 300;
+/// Delay for caching a node info response in WASM runtime
+#[cfg(target_family = "wasm")]
+pub(crate) const CACHE_NETWORK_INFO_TIMEOUT_IN_SECONDS: u32 = 1;
```

Comment:
Bech32, network id... never changes, only the milestone timestamp which we use in `get_time_checked()` and there we just need to be somehow correct (currently it's +-5 minutes), so should be fine with 60 seconds?
Only 1 will still send many more requests than needed

Reply:
idk its hard to test the cache in an automated way